lib/jsdom/living/nodes/HTMLInputElement-impl.js

```diff
@@ -200,7 +200,14 @@ class HTMLInputElementImpl extends HTMLElementImpl {
         fireAnEvent("input", this, undefined, { bubbles: true });
         fireAnEvent("change", this, undefined, { bubbles: true });
       }
-    } else if (form && (this.type === "submit" || this.type === "image")) {
+    } else if (form && this.type === "image") {
+      // https://html.spec.whatwg.org/multipage/input.html#image-button-state-(type=image):input-activation-behavior
+
+      // At this point the dispatch flag is false, so offsetX/Y reflect pageX/Y, which is not what we want.
+      // Instead we use _paddingEdgeRelativeX/Y, since they reflect the actual selected coordinates.
+      this._selectedCoordinate = { x: event._paddingEdgeRelativeX, y: event._paddingEdgeRelativeY };
```

Comment:
Perhaps there's a better way? The challenge here is that `_activationBehavior` is our best hook to know a click happened, but it happens after the dispatch flag is unset.

This allows us to follow the spec, but perhaps this is all overkill, since we don't implement layout and it would probably be ok just to return `(0, 0)` 🤷  ... regarding CSSOM and offsetX/offsetY it looks like browsers [don't even consistently implement](https://github.com/w3c/csswg-drafts/issues/1297) the non-dispatch branch ([step 2](https://drafts.csswg.org/cssom-view/#dom-mouseevent-offsetx))